testing/diff.go

```diff
@@ -57,6 +58,10 @@ func (*differ) Event(expected, actual Event) string {
 	return cmp.Diff(expected, actual)
 }
 
+func (*differ) ApplyRef(expected, actual ApplyRef) string {
+	return cmp.Diff(expected, actual, NormalizeApplyConfiguration)
```

Comment:
normalizing the ApplyConfiguration as json is technically correct, but I wonder if the usability cost is too high as all other type info on the object being compared is lost.

A user can define a custom differ to change the behavior, but I'm wondering if the default should be to not apply the NormalizeApplyConfiguration transformer.